deploy/bundle/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -2756,6 +2756,13 @@ spec:
                   initialDelaySeconds: 5
                   periodSeconds: 10
                 resources: {}
+                securityContext:
```

Comment:
Can you pls check why these changes are not getting generated for the metrics exporter?

Reply:
I am not sure, But I just found out metrics exporter actually needs privileged access. Why it needs has been explained [here](https://github.com/red-hat-storage/ocs-operator/pull/1638/commits/6d081d662abfdf722f98fadbb63511e008f42b1c). So I am reverting the changes for the metrics-exporter.